build-system/common/utils.js

```diff
@@ -35,6 +35,14 @@ function buildMinifiedRuntime() {
   execOrDie(`gulp dist --fortesting --config ${argv.config}`);
 }
 
+/**
+ * Cleaned and builds unminified runtime
+ */
+function buildUnminifiedRuntime() {
+  execOrDie('gulp clean');
+  execOrDie('gulp build --fortesting');
```

Comment:
For future proofing, consider including `--config ${argv.config}` in this command. Should be a no-op if `--config` is not passed in, and will serve the `canary` config if `--config canary` is passed in.